public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,171 @@
+# dvc.api.open()
+
+[Context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+to open a tracked file.
+
+## Definition
+
+### Signature
+
+```py
+open(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type is a
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+No exceptions are raised by this function directly.
+
+## Description
+
+This function is analogous to the
+[`open()`](https://docs.python.org/3/library/functions.html#open) Python
+builtin, but for files tracked in <abbr>DVC projects</abbr> (by DVC or Git).
+However, it may only be used as a _context manager_ (using the `with` keyword as
+shown in the **Examples** below). There is no `close()`.
+
+> See also `dvc.api.read()` for a shorthand way to read the complete contents of
+> a tracked file.
+
+💡 Note that `dvc.api.open()` is able to
+[stream](https://docs.python.org/3/library/io.html) the file directly from
+**most**
+[remote types](/doc/command-reference/remote/add#supported-storage-types)
+(local, S3, Azure, GCP, OSS, SSH, HDFS). Otherwise, the file is downloaded
+regularly into a temporary local path before the file object is made available.
+
+> This has similar uses as the `dvc get` and `dvc import` CLI commands.
```

Comment:
this is similar to `dvc get`, not `dvc import` I would say

Reply:
True. Updated.